### ISSUE TITLE
Coletar informações ocupação dos leitos por estado

### ISSUE THREAD

**Issue by capyvara** [turicas/covid19-br#117]
Aparentemente o MS não centraliza esses dados, parece que estão disponíveis nas secretarias.

Os totais da para pegar daqui se necessário: 
https://covid-insumos.saude.gov.br/paineis/insumos/painel_leitos.php

**Comment by turicas** [comment]
https://superset.plataformatarget.com.br/superset/dashboard/72/

**Comment by capyvara** [comment]
SP: https://www.seade.gov.br/coronavirus/ (tem CSV e repo abaixo)
PR: http://www.saude.pr.gov.br/modules/conteudo/conteudo.php?conteudo=3507 (tem CSV)
PA: https://www.covid-19.pa.gov.br/#/
RS: http://ti.saude.rs.gov.br/covid19/
MA: http://www.saude.ma.gov.br/boletins-covid-19/
AL: http://www.alagoascontraocoronavirus.al.gov.br/

**Comment by capyvara** [comment]
Eu acho que esses dados de baixa densidade daria para resolver com voluntários colocando os dados manualmente diariamente, não?

**Comment by endersonmaia** [comment]
> Eu acho que esses dados de baixa densidade daria para resolver com voluntários colocando os dados manualmente diariamente, não?

sim, eu estou tentando compilar os de AL, desde que eles iniciaram a disponibilizar

ainda na dúvida de quais são os tipos de leitos que devo catalogar

no boletim de Alagoas tem :

 -  UTI Adulto
 -  UTI Pediátrica
 -  Leito Clínico Adulto
 -  Leito Clínico Pediátrico
 -  U Intermediária ("U" deve ser de Unidade Intermediária)

Existem umas nomenclaturas não padronizadas, como "COVID + HIV" q penso considerar Clínico Adulto, e "COVID + TB" (tuberculose?) q penso em considerar Clínico Adulto tb.

Existe algum padrão e leitos para definirmos um modelo de dados para coleta ?

**Comment by endersonmaia** [comment]
@capyvara tomarei a liberdade de editar tua issue, a ideia é tentarmos mapear qual nível de detalhe temos em cada estado que divulga estes dados, ok ?

vou iniciar com os links que vc já coletou

**Comment by endersonmaia** [comment]
algo a se observar, é deixar claro qual a cobertura deste levantamento de dados

são considerados os leitos disponíveis no TOTAL, entre rede Pública e Privada ?

No boletim do MA tem a observação : 

> ***Taxa de ocupação relativa aos leitos SUS disponíveis na rede SES/MA

Em AL já tem a seguinte observação :

> REDE PÚBLICA E CONTRATUALIZADOS

**Comment by fernandascovino** [comment]
a SES RJ só mostra a média mensal de ocupação dos leitos a cada ano (mais recente: 2019): http://www.rio.rj.gov.br/web/sati/media-mensal-de-leitos-uti-mun-rio-de-janeiro

Aparentemente é só um espelho do DataSUS... Na página também indicam o painel do MS: https://covid-insumos.saude.gov.br/paineis/insumos/painel_leitos.php

**Comment by augusto-herrmann** [comment]
O DF disponibiliza esses dados diariamente em um documento PDF. A URL do documento PDF varia, mas o link para ele está na página "[Transparência na Saúde](http://www.saude.df.gov.br/transparencia-na-saude/)", sob a seção UTI. Lá está inclusa a informação se o leito é para Covid ou não. Exemplo de 4/5/2020:

* [Situação dos Leitos de UTI – 04/05/2020](http://www.saude.df.gov.br/wp-conteudo/uploads/2019/09/Relatorio_Situacao_Leitos_UTI-V3-2020-05-04.pdf)

As informações são detalhadas sobre cada leito individual, incluindo a informação de data e hora de ocupação do leito de UTI.

Para contabilizar os leitos sobre Covid seria um pouco trabalhoso, pois a pessoa teria que procurar na lista e somar ela mesma. É oferecida totalização apenas para os leitos de UTI em geral, mas não uma totalização sobre a Covid.

Não consegui encontrar informações sobre leitos clínicos ocupados/disponíveis para Covid-19.

**Comment by endersonmaia** [comment]
> a SES RJ só mostra a média mensal de ocupação dos leitos a cada ano (mais recente: 2019): http://www.rio.rj.gov.br/web/sati/media-mensal-de-leitos-uti-mun-rio-de-janeiro
> 
> Aparentemente é só um espelho do DataSUS... Na página também indicam o painel do MS: https://covid-insumos.saude.gov.br/paineis/insumos/painel_leitos.php

vc informou link do município, os dados devem ser coletados na SES do RJ (http://saude.rj.gov.br), de qq forma, tb não encontrei por lá

**Comment by endersonmaia** [comment]
sobre os tipos de leitos, podemos tentar inferir quais os tipos de leitos informados no boletim com o cadastro do CNES

http://cnes2.datasus.gov.br/Mod_Ind_Tipo_Leito.asp

identifiquei os código 51 e 52 como sendo UTI específico para COVID-19

```
51 | UTI II ADULTO - COVID-19
52 | UTI II PEDIATRICA - COVID-19
```

aqui no repositório tem um script que baixa todas as tabelas dos CNES com vários meses de competência

https://github.com/turicas/covid19-br/blob/master/scripts/cnes.sh

assim daria também pata informar qual unidade de saúde usando o código do estabelecimento, e até geolocalização

**Comment by ramiroluz** [comment]
Eu estou tentando manter uma planilha pessoal com os dados do boletim de curitiba, tem o número de leitos ocupados somados (pub+privado). Em alguns dias sai o percentual de ocupação também, mas não sei como é calculado e alguns dias não sai. Se for o caso me disponho a fazer um crawler para o site da prefeitura de curitiba e/ou disponibilizar a planilha(ainda preciso atualizar alguns boletins).

**Comment by ecrisol** [comment]
Vários estados já estão disponibilizando em painel os dados: 
DF - https://salasit.saude.df.gov.br/publicos-leitos-covid-internacao/; 
RS - https://covid.saude.rs.gov.br/
MG - https://www.saude.mg.gov.br/coronavirus/painel
ES - https://coronavirus.es.gov.br/leitos-uti
GO - https://www.saude.go.gov.br/coronavirus
RO - http://covid19.sesau.ro.gov.br/Home/Leitos
CE - tem como retirar dos microdados gerais - lá tem evolução dos casos - onde informa UTI e Internado; O mesmo é possível para os dados de pernambuco

**Comment by ecrisol** [comment]
PB também de CSv para leitos; um csv para uti, outro para internações;
